src/extension.ts

```diff
@@ -412,9 +444,9 @@ class Workspace {
       debugLog.appendLine(`error: Infracost cmd error trace ${error}`);
 
       if (init) {
-        vscode.window.showErrorMessage(`Could not run the infracost cmd in the ${path} directory. If this is a multi-project workspace please try opening just a single project. If this problem continues please open an issue here: https://github.com/infracost/vscode-infracost.`);
+        vscode.window.showErrorMessage(`Could not run the infracost cmd in the ${path} directory. This is likely becuase of a syntax error, or invalid project. See the Infrcost Debug output tab for more information. Go to View > Output & select "Infracost Debug" from the dropdown. If this problem continues please open an issue here: https://github.com/infracost/vscode-infracost.`);
```

Comment:
```suggestion
        vscode.window.showErrorMessage(`Could not run the infracost cmd in the ${path} directory. This is likely because of a syntax error or invalid project. See the Infracost Debug output tab for more information. Go to View > Output & select "Infracost Debug" from the dropdown. If this problem continues please open an issue here: https://github.com/infracost/vscode-infracost.`);
```